### PR TITLE
feat(DIA-109): add represented by section on Artist screen

### PR DESCRIPTION
**Pull request by araujobarret**
This PR resolves [DIA-109] 

### Description

Adds represented by horizontal list to the Artist screen.

iOS

https://github.com/artsy/eigen/assets/15792853/cb8c422b-f543-44f2-ac2e-303e7193d193

Android

https://github.com/artsy/eigen/assets/15792853/b95dcb64-3c9b-4e3d-b78b-30e0eedff451

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- add represented by rails to the artist screen - araujobarret

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[DIA-109]: https://artsyproduct.atlassian.net/browse/DIA-109?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/Artist/Artist.tsx [hunk 1]
```diff
@@ -329,8 +329,7 @@ const ArtistSkeleton: React.FC = () => {
             <Spacer y={2} />
             <Join separator={<Spacer y={0.5} />}>
               <SkeletonText variant="lg">Artist Name Artist Name</SkeletonText>
-              <SkeletonText variant="xs">American, b. 1945</SkeletonText>
-              <SkeletonText variant="xs">40 Works, 45 Followers</SkeletonText>
+              <SkeletonText variant="lg">American, b. 1945</SkeletonText>
```
Comment (araujobarret) [inline src/app/Scenes/Artist/Artist.tsx:332]: small adjust to skeleton to reflect what we really have

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (add represented by rails to the artist screen - araujobarret)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 1e502f472005a0b2f4948116d0d659b686baccfa
</p>

**Review by MounirDhahri — APPROVED** [review]
Nice!

**Review by gkartalis — APPROVED** [review]
🛳️

**Review by egdbear — APPROVED** [review]
🚀

**Pull request merged**